website/docs/experimental/rerun.md

```diff
@@ -0,0 +1,92 @@
+---
+id: rerun
+title: Re-run a job from previous config
+sidebar_label: Re-run 
+---
+
+import {ExampleGithubLink} from "@site/src/components/GithubLink"
+
+<ExampleGithubLink text="Example application" to="examples/experimental/rerun"/>
+
+:::caution
+This is an experimental feature. Please read through this page to understand what is supported.
+:::
+
+We use the example app linked above for demonstration. To save the configs for re-run, first use the experimental
+Hydra Callback for saving the job info:
+
+
+```yaml title="config.yaml"
+hydra:
+  callbacks:
+    save_job_info:
+      _target_: hydra.experimental.pickle_job_info_callback.PickleJobInfoCallback
+```
+
+
+
+
+```python title="Example function"
+@hydra.main(config_path=".", config_name="config")
+def my_app(cfg: DictConfig) -> None:
+    log.info(f"output_dir={HydraConfig.get().runtime.output_dir}")
+    log.info(f"cfg.foo={cfg.foo}")
+```
+
+
+Run the example app:
+```commandline
+[2022-03-16 14:51:30,905][hydra.experimental.pickle_job_info_callback][INFO] - Saving job configs in /Users/jieru/workspace/hydra/examples/experimental/outputs/2022-03-16/14-51-30/.hydra/config.pickle
```

Comment:
```suggestion
$ python my_app.py
[2022-03-16 14:51:30,905][hydra.experimental.pickle_job_info_callback][INFO] - Saving job configs in /Users/jieru/workspace/hydra/examples/experimental/outputs/2022-03-16/14-51-30/.hydra/config.pickle
```
I'd suggest to add the python command for consistency with the following commandline block.